src/reanimated2/hook/useSharedValue.ts

```diff
@@ -3,11 +3,16 @@ import { cancelAnimation } from '../animation';
 import { SharedValue } from '../commonTypes';
 import { makeMutable } from '../core';
 
-export function useSharedValue<T>(init: T): SharedValue<T> {
-  const ref = useRef<SharedValue<T>>(makeMutable(init));
+export function useSharedValue<T>(
+  init: T,
+  needSynchronousReadsFromReact = false
```

Comment:
```suggestion
  needsSynchronousReadsFromReact = false
```
BTW `readableFromReact` is way shorter